organization/_posts/2017-10-26-startup.md

```diff
@@ -0,0 +1,99 @@
+    - We would also like to make sure that all major experiments, section editors and regional representatives can sign off. 
+    - Created a [google doc](https://docs.google.com/document/d/1tBXwlNnQsxxZA3gVS1_KSpa8wRXGyk250EIsJwJ2T34/edit) to collect signatures
+    - Place to This has already been created: 
+  - Pete was in touch (via Kyle) with an editor of the Symmetry magazine. We should pitch the CWP here and to CERN Courier.
+
+- #### Simulation
+  - Documents are quite fixed, should be good to go into arXiv once the author list is done.
+
+- #### Machine Learning
+  - No news. Document seems in good shape.
+
+- #### Software Development
+  - Just need to add references, then ready for arXiv.
+
+- #### Software Trigger and Event Reconstruction
+  - ATLAS are delaying approval of the dedicated document on real-time analysis for 2-3 weeks (Cate knows details why) which we reference, so we will delay arXiv submission to wait for them. 
+
+- #### Data Access, Organisation and Management
+  - Need feedback from Andy H - should push a bit more with a deadline.
+
+- #### Data and Software Preservation
+  - No news this week, still a few comments to process but nothing major. Waiting for Mike to do it.
+
+- #### Data Analysis and Interpretation
+  - No news. Seems to be in good shape, but may require some effort to get it more coherent with others. Also some work to make the roadmap a bit more precise would be good.
+
+- #### Workflow and Resource Management
+  - Still waiting a draft from Kaushik.
+
+- #### Visualization
+  - Ric tied up a lot of loose ends last week. Still needs to be tidied up more before arXiv.
+
+- #### Event/Data Processing Frameworks
+  - Ready for distribution, signoff from main author tomorrow.
+
+- #### Careers, Staffing and Training
+  - Ready for arXiv. Dario thanks Michel for the excellent summary he provided
+
+- #### Facilities and Distributed Computing
+  - No news.
+
+- #### Conditions Access
+  - No news.
+
+- #### Generators
+  - Need to chase this up with Taylor. We do need to push MCNet here because it was bad that this was missing from the first roadmap draft. Liz will make sure that CMS contribute their view.
+
+- #### Security
+  - Romain Wartel asked if there was still time to add something. They do seem to be a close community so they may be able to converge on some input quite quickly. Michel will follow up with Hannah Short and Mine. 
+
+## Activity updates
+
+### Licensing
+  - Graeme and Benedikt were approached by LHCb about license situation. They are thinking about moving the experiment stack to GPLv3. To allow for using as many external libraries as possible. They would like to know how much of existing LCG stack is v3 compatible yet. Our only point of concern for making LHCb GPLv3 is the possibility of factorizing parts out of LHCb into experiment-independent libraries. They would better be LGPL or Apache.
```

Comment:
"They would better be LGPL or Apache" -> "Those pieces would better under LGPL or Apache"

Reply:
Done